docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tools for porting to .NET Core
+description: Learn about porting C++/CLI projects to .NET Core
+author: mjrousos
+ms.author: 
+ms.date: 12/12/2019
+---
+
+# How to port a C++/CLI project to .NET Core
+
+Beginning with .NET Core 3.1 and Visual Studio 2019 16.4, C++/CLI projects can be updated to target .NET Core. This allows Windows desktop applications which may use C++/CLI projects as an interop layer between managed and native code to be ported to .NET Core. This article describes how to port C++/CLI projects from .NET Framework to .NET Core 3.1.
```

Comment:
Is there a good link for `C++/CLI` to add here?

Reply:
I'll add a link.